tests/test_dunders.py

```diff
@@ -624,9 +624,8 @@ def test_init(self, slots, frozen):
         with pytest.raises(TypeError) as e:
             C(a=1, b=2)
 
-        assert (
+        assert e.value.args[0].endswith(
```

Comment:
Interesting. What does it start with now?

Reply:
The class names. So in this case `C.`.